operators/src/source/ogr_source/dataset_iterator.rs

```diff
@@ -48,14 +49,17 @@ impl OgrDatasetIterator {
         query_rectangle: &VectorQueryRectangle,
         attribute_filters: Vec<AttributeFilter>,
     ) -> Result<OgrDatasetIterator> {
+        let adjusted_filters =
```

Comment:
also: could you add a test where a column is renamed?

Reply:
I added a test with renaming: 863988315ee64ee2597350dc330fa34e212c1898 (Sorry, wrong commit message)
Moreover, I vote to keep the reverse mapping in this place because:
- In order to do the mapping earlier, we would need to request the loading_info during initialization which is quite expensive for some sources (e.g., NFDI). When keeping it here, the loading info is present since it is requested by the query processor anyway.
- The filters are passed from Source to InitializedSource to QueryProcessor. I think it is more sound to formulate the filters on the renamed attributes in all those places and do the re-mapping just for gdal.